cmd/lncli/commands.go

```diff
@@ -1539,47 +1517,31 @@ mnemonicCheck:
 		// want to use, we'll generate a fresh one with the GenSeed
 		// command.
 		fmt.Println("Your cipher seed can optionally be encrypted.")
-		fmt.Printf("Input your passphrase if you wish to encrypt it " +
+
+		instruction := "Input your passphrase if you wish to encrypt it " +
 			"(or press enter to proceed without a cipher seed " +
-			"passphrase): ")
-		aezeedPass1, err := terminal.ReadPassword(int(syscall.Stdin))
+			"passphrase): "
+		aezeedPass, err = capturePassword(instruction, true,
```

Comment:
similar wrapping comment:
```golang
aezeedPass, err = capturePassword(
        instruction, true, func(password []byte) error {
	        return nil
	},
)
```
also since we don't use the password argument, we can replace `password` with `_` if space is tight :)